src/template.py

```
@@ -0,0 +1,420 @@
+import pathlib
+import subprocess
+import logging
+import os
+import yaml
+import time
+import threading
+import datetime
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)